#system parameters
P_x: 1.0
P_y: 1.0
P_z: 1.0
P_roll: 1.0
P_pitch: 1.0
P_yaw: 1.0

D_x: 1.0
D_y: 1.0
D_z: 1.0
D_roll: 1.0
D_pitch: 1.0
D_yaw: 1.0

n_tether: 4

tether_configuration:
  '0':
      mounting_pos: {x: 0.21, y: 0.0, z: 0.0}
      anchor_position: {x: 5.21, y: 0.0, z: 0.0}
  '1':
      mounting_pos: {x: 0.0, y: -0.21, z: 0.0}
      anchor_position: {x: 0.0, y: -5.21, z: 0.0}
  '2':
      mounting_pos: {x: -0.21, y: 0.0, z: 0.0}
      anchor_position: {x: -5.21, y: 0.0, z: 0.0}
  '3':
      mounting_pos: {x: 0.0, y: 0.21, z: 0.0}
      anchor_position: {x: 0.0, y: 5.21, z: 0.0}
